app/config/capistrano/stages/staging.rb

```diff
@@ -14,3 +14,15 @@
 
 SSHKit.config.command_map[:composer] = "#{fetch :php_bin} #{shared_path.join("composer.phar")}"
 SSHKit.config.command_map[:php] = fetch(:php_bin)
+
+namespace :composer do
+    desc 'Install the vendors'
+    task :install_vendors do
```

Comment:
Hmm, volgens mij kunt ge beter php_bin correct instellen of composer mappen

Reply:
Iets als https://github.com/tijsverkoyen/capistrano-forkcms/blob/5fa1bafda27fca8fc75cb9299dbf9d01553bb876/lib/capistrano/tasks/configure.rake#L12

Reply:
@tijsverkoyen maar dat zou die toch al moeten gebruiken? @Katrienvh had daar nog problemen mee volgens @StijnVrolijk

Reply:
Ofwel snap ik uw probleem niet, ofwel probeert ge iets raar op te lossen.
Omschrijf in de PR maybe eens het probleem deftig.

Reply:
composer install zelf werkt wel maar in de scripts staat gewoon php bin/console forkcms:cache:clear bijvoorbeeld, daar wordt dus de gewone php versie gebruikt wat voor problemen zorgt omdat daar ook bv 7.1 nogig is

Reply:
Niet logischers om geen scripts uit te voeren, en die aan te roepen vanuit Capistrano als die al nodig zijn.

Reply:
dan moet je weer extra stuff configureren